src/keylessBackup/SignInWithEmail.tsx

```diff
@@ -85,7 +86,7 @@ function SignInWithEmail({ route }: Props) {
         style={styles.button}
         icon={<GoogleIcon />}
         iconMargin={12}
-        touchableStyle={loading ? undefined : styles.buttonTouchable}
+        touchableStyle={[styles.buttonTouchable, !loading && { justifyContent: 'flex-start' }]}
```

Comment:
bit of a hack to get these styles on Button, checking with Nitya on if this is going to be a more common style that can be its own button type, in which case will make a follow up PR to clean this up.